driver/controller.go

```diff
@@ -257,7 +260,8 @@ func (s controllerService) GetCapacity(ctx context.Context, req *csi.GetCapacity
 			return &csi.GetCapacityResponse{AvailableCapacity: 0}, nil
 		case nil:
 		default:
-			return nil, status.Error(codes.Internal, err.Error())
+			ctrlLogger.Error(err, "get capacity failed")
+			return &csi.GetCapacityResponse{AvailableCapacity: 0}, nil
```

Comment:
GetCapacityByTopologyLabel calls getNodes and the method would return some real error.
Would you revise the code to handle errors more precisely?